src/main/java/tutoraid/model/student/InitialStudent.java

```diff
@@ -0,0 +1,62 @@
+package tutoraid.model.student;
```

Comment:
InitialStudent that must be combined with LessonBook to produce Student (which contains Lesson objects)